src/deploy/mod.rs

```diff
@@ -176,76 +234,74 @@ async fn recalculate_mode_scores(
         None => "".to_string(),
     };
 
+    let beatmap_md5s: Vec<(String,)> = sqlx::query_as(&format!(
+        "SELECT DISTINCT beatmap_md5 FROM {} WHERE completed IN (2, 3) AND play_mode = ? {}",
+        scores_table, mods_query_str,
```

Comment:
does rust have any simple query builders? not really a fan of orms but query builders are nice for things like this `mode_query_str`

Reply:
cc @infernalfire72

Reply:
there's a few although i don't remember their names from the top of my head, i'll have a look around

Reply:
looked more into this and i can't really find any query builders that aren't headed too deep in the ORM direction so i think i'm just going to live with this for now. other problem is the libraries i found are also not very mature... average rust tbh